tests/copp/copp_utils.py

```diff
@@ -0,0 +1,118 @@
+"""
+Helpful utilities for writing tests for the COPP feature.
+
+Todo:
+    Refactor ptfadapter so it can be leveraged in these test cases.
+"""
+
+DEFAULT_NN_TARGET_PORT = 3
+
+_REMOVE_IP_SCRIPT = "scripts/remove_ip.sh"
+_ADD_IP_SCRIPT = "scripts/add_ip.sh"
+_UPDATE_COPP_SCRIPT = "copp/scripts/update_copp_config.py"
+
+_BASE_COPP_CONFIG = "/tmp/00-copp.config.json"
+_SWSS_COPP_CONFIG = "swss:/etc/swss/config.d/00-copp.config.json"
+_TEMP_COPP_CONFIG = "/tmp/copp_config.json"
+
+_PTF_NN_TEMPLATE = "copp/templates/ptf_nn_agent.conf.ptf.j2"
```

Comment:
should the template and all helper files go under <test dir>/files/

Reply:
I'm still a little confused about this, maybe @neethajohn can give us some pointers?

Reply:
helper func under files/. if you have too many templates I think they can have their own templates/ folder

Reply:
Moved them to the common folder. I kept the script were it is since it's pretty specific to the COPP tests, can move if you guys would prefer. :)